recipes/sparse-neighbors-search/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: sparse-neighbors-search
+  version: 0.2.3
+
+source:
+  url: https://github.com/joachimwolff/minHashNearestNeighbors/archive/v0.2.3.tar.gz
+  md5: 49b3f9442b4caad82f8285c830296490
+
+build:
+  skip: True  # [not py27]
+  number: 0
+
+requirements:
+  build:
+    - python 2.7.12
```

Comment:
the version number is not needed here, as you skip the build in the build section